docs/AddressSanitizing.md

```diff
@@ -0,0 +1,31 @@
+# Address Sanitizing
+
+*AddressSanitizer* (aka ASan) is a very useful library for detecting memory-related issues for C/C++. It consists of a compiler instrumentation module and a run-time library. The typical performance penalty for address sanitizing is 2x.
```

Comment:
Not sure it makes sense to add editorial comments like "very useful", but that is just my view.

Reply:
yeah, agreed